app/javascript/app/components/my-climate-watch/viz-creator/components/charts/legend/legend-component.jsx

```diff
@@ -19,15 +20,14 @@ const LegendComponent = ({ className, data = [], theme }) => (
           />
         ))}
     </ul>
-    {data &&
-      data.logo && (
-        <div className={theme.legendLogoContainer}>
-          <div className={theme.legendLogoTitle}>Data provided by:</div>
-          <a href={data.modelUrl} target="_blank" className={theme.legendLogo}>
-            <img id="logoImage" src={`https:${data.logo}`} />
-          </a>
-        </div>
-      )}
+    {hasLogo && (
+      <div className={theme.legendLogoContainer}>
```

Comment:
maybe extract it to another component `Logo`?

Reply:
This is not really part of the PR. It was just some variable to avoid lint warnings. Is true that sometimes we could extract some components if they could be reused somewhere else. Normally this is our main reason to extract. It could also make the code more readable in big files. In this case, I'm not sure if this is reused somewhere else.

Reply:
I see but I would still extract this to another component even if it's only for better readability, this way you'd have a one liner: 
`hasLogo && <Logo props=someProps />`
Also this correspons to component's single responsibility rule.